src/Turbo/src/Bridge/Mercure/Broadcaster.php

```diff
@@ -40,17 +40,15 @@ final class Broadcaster implements BroadcasterInterface
      */
     public const TOPIC_PATTERN = 'https://symfony.com/ux-turbo/%s/%s';
 
-    private $name;
-    private $hub;
-
-    /** @var ExpressionLanguage|null */
+    /**
+     * @var ExpressionLanguage|null
+     */
     private $expressionLanguage;
```

Comment:
Can't we type it without PHPDoc? (note the assignment)
```suggestion
    private ExpressionLanguage|null $expressionLanguage = null;
```

Reply:
Well spoted. I set the value in the constructor.